### PR TITLE
Fixed text files parsing

### PR DESCRIPTION
**Pull request by SweetyAngel**
Just a check if a file is text

### CODE REVIEW

File: Scripts/lua_singleton.gd [hunk 1]
```diff
@@ -417,7 +418,17 @@ func _splitstr(input: String, separator: String):
 func _trim(input: String):
 	return input.strip_edges()
 
-func setup_extension(extension):
+func setup_extension(file):
+	if len(file.split(".")) == 1:
+		editor.warn("[color=yellow]WARNING[/color]: File has no extension.\nGriddyCode can't recognize it's language.\nFile will be parsed as raw text.")
```
Comment (RealPacket) [inline Scripts/lua_singleton.gd:423]:
```suggestion
		editor.warn("[color=yellow]WARNING[/color]: File is plain text.\nGriddyCode can't recognize its language.\nFile will be parsed as raw text.")
```

File: Scripts/lua_singleton.gd [hunk 2]
```diff
@@ -417,7 +418,17 @@ func _splitstr(input: String, separator: String):
...
+		return
+
+	var extension = file.split('.')[-1]
+
+	if extension in text_extensions:
+		editor.warn("[color=yellow]WARNING[/color]: File is text file, not code.")
+		return
```
Comment (RealPacket) [inline Scripts/lua_singleton.gd:430]:
```suggestion
	if extension == "txt" :
		editor.warn("[color=yellow]WARNING[/color]: File is text file, not code.")
		return
```

There's literally only 1 extension in text_extensions. Why do we need an array right now?

**Review by RealPacket — CHANGES_REQUESTED** [review]
I think that it would be better if we merged #111 (disclaimer: my PR) because it actually fixes the core issue.
In this case, the core issue is that GriddyCode doesn't clear language data (syntax, IDE comments (my PR requires you to open the menu twice, probably will fix it later), etc.)

**Comment by SweetyAngel** [comment]
I will close this PR because I agree that your PR need to be merged

**Comment by Real-Packet** [comment]
GitHub decided to suspend my account, I think I have the source code for the PR somewhere on my other disk

**Pull request closed without merging**